src/configfetcher.cpp

```diff
@@ -71,6 +71,32 @@ ConfigFetcher::ConfigFetcher(const string& sdkKey, shared_ptr<ConfigCatLogger> l
                 return !closed;
             }
     });
+
+    if(options.sslOptions)
+    {
+        // TODO: due to a bug in CPR requirest the cpr::SslOptions object to be kept alive. to discuss with Peter
+        static cpr::SslOptions sslOpts;
+        if(!options.sslOptions->caBuffer.empty()) {
+            std::string temp = options.sslOptions->caBuffer;
+            sslOpts.SetOption(cpr::ssl::CaBuffer {std::move(temp)});
+        }
+
+        if(options.sslOptions->verifySSL) {
+            const bool shouldVerify = *options.sslOptions->verifySSL;
+
+            if(!shouldVerify)
+            {
+                // TODO: Check wording + ask if we need a warning number
+                LOG_WARN(9999)  << "Peer verification will be disabled. We highly suggest using HTTPS for production grade apps.";
```

Comment:
If disabling of peer verification is too dangerous, we can get ride of this warning.

Reply:
option was removed